### PR TITLE
Solve serialization depth problems

### PR DESCRIPTION
**Pull request by vogdb**

### CODE REVIEW

**Comment by vogdb** [comment]
@tomaszdurka This is a very important fix. Please review.

**Comment by tomaszdurka** [comment]
As discussed put directly ids of plugin, session, connection.

File: lib/janus/plugin/abstract.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ PluginAbstract.prototype._isSuccessResponse = function(response) {
 };
 
 PluginAbstract.prototype.onRemove = function() {
-  serviceLocator.get('logger').info('Removed plugin', {plugin: this});
+  serviceLocator.get('logger').info('Removed plugin', {plugin: this.id});
```
Comment (tomaszdurka) [inline lib/janus/plugin/abstract.js]: Let's manually pass session and connection here too!

Comment (tomaszdurka) [inline lib/janus/plugin/abstract.js]: Or at least session.

File: lib/janus/plugin/audio.js [hunk 2]
```diff
@@ -56,7 +56,7 @@ PluginAudio.prototype.onJoin = function(message) {
     if (plugin._isSuccessResponse(response) && 'joined' == response['plugindata']['data']['audioroom']) {
       var channel = new Channel(response['plugindata']['data']['uid'], message['body']['id'], message['body']['channel_data']);
       plugin.stream = Stream.generate(channel, plugin);
-      serviceLocator.get('logger').info('Added stream for plugin', {stream: plugin.stream, plugin: plugin});
+      serviceLocator.get('logger').info('Added stream for plugin', {stream: plugin.stream.id, plugin: plugin.id});
```
Comment (tomaszdurka) [inline lib/janus/plugin/audio.js]: Same goes here (session and possibly connection) and for all below.

**Comment by tomaszdurka** [comment]
lgtm

**Pull request merged**